test/generator.cpp

```diff
@@ -18,7 +18,7 @@ void generate() {
     using namespace autocheck;
     std::clog << gen(i);
   }
-  std::clog << std::endl;
+  std::clog << '\n';
```

Comment:
I was under the impression `'\n'` was not portable, but it looks like it is for [text mode](http://stackoverflow.com/questions/1911018/what-does-stdendl-represent-exactly-on-each-platform).